client/web/src/cody/management/subscription/new/CodyProCheckoutForm.tsx

```diff
@@ -14,99 +18,47 @@ export const CodyProCheckoutForm: React.FunctionComponent<{
     stripePromise: Promise<Stripe | null>
     customerEmail: string | undefined
 }> = ({ stripePromise, customerEmail }) => {
-    const [clientSecret, setClientSecret] = useState('')
-    const [errorDetails, setErrorDetails] = useState('')
-    const [urlSearchParams] = useSearchParams()
-
-    // Optionally support the "showCouponCodeAtCheckout" URL query parameter, which is present
+    // Optionally support the "showCouponCodeAtCheckout" URL query parameter, which if present
```

Comment:
Idea to further avoid confusion

```suggestion
    // Optionally support the "showCouponCodeAtCheckout" URL query parameter, which, if present,
```

Reply:
Done.